README.md

```diff
@@ -1,12 +1,24 @@
 # DuckDuckGo Android
 
-Welcome to our android application.  While it is still too early to accept contributions, we are excited to engage the community in development and will open up this project to contributions in the future.
+Welcome to our android application. We are excited to engage the community in development see [CONTRIBUTING.md](https://github.com/duckduckgo/Android/blob/develop/CONTRIBUTING.md).
```

Comment:
grammar: should probably have something like a dash or semicolon between "development" and "see"

Reply:
As per other comment, would a relative link work better here?